src/plugin/featureaction/ui/editfeatureaction.js

```diff
@@ -120,7 +163,8 @@ plugin.im.action.feature.ui.EditFeatureActionCtrl.prototype.buildStylePreview =
   var feature = new ol.Feature();
   var geometry = null;
 
-  this.styleCanvas = /** @type {HTMLCanvasElement} */ (document.createElement('canvas'));
+  this.styleCanvas = /** @type {HTMLCanvasElement} */ (document.createElement(
```

Comment:
Did your IDE auto-format this file for 80 columns? There are a lot of indentation changes that we don't want (we allow up to 120 columns).

Reply:
I have it set for wrapping at 120